packages/opentelemetry-api/src/api/context.ts

```diff
@@ -76,6 +96,20 @@ export class ContextAPI {
    * @param context context to bind to the event emitter or function. Defaults to the currently active context
    */
   public bind<T>(target: T, context: Context = this.active()): T {
-    return this._contextManager.bind(target, context);
+    return this._getContextManager().bind(target, context);
+  }
+
+  private _getContextManager(): ContextManager {
+    if (!(global as any)[GLOBAL_CONTEXT_MANAGER_API_KEY]) {
+      return NOOP_CONTEXT_MANAGER;
+    }
+
+    return (global as any)[GLOBAL_CONTEXT_MANAGER_API_KEY](API_VERSION);
```

Comment:
Could we create a global ref var in this file to remove these `as any`?
```ts
const _global = global as typeof global & { [key: Symbol]: whatever };
```
```ts
...
  return (global as any)[GLOBAL_CONTEXT_MANAGER_API_KEY](API_VERSION);
...
```
WDYT?

Reply:
Updated. PTAL